jobs/keepalived/spec

```diff
@@ -16,6 +16,17 @@ templates:
   bin/get_sf_broker_status.sh.erb: 	bin/get_sf_broker_status.sh
   config/keepalived.conf.erb: 		config/keepalived.conf
 
+provides:
+- name: keepalived
+  type: keepalived
+  properties:
+  - status_script
+
 consumes:
 - name: broker
   type: broker
+
+properties:
+  status_script:
+    description: "Script to check current node is master or backup"
+    default: "/var/vcap/jobs/keepalived/bin/get_keepalived_state.sh"
```

Comment:
This shouldn't be a default value, right? If the setup is non-HA, then the value should be unavailable.

Reply:
On a second thought though, I think exporting a default location could be fine. As the module consuming this variable could instead check if the file exists (true for HA and false for non-HA) and handle accordingly.

Reply:
why is this even being exported? who needs this property?